src/components/RequestModal/AdvancedRequester/index.tsx

```diff
@@ -18,12 +18,10 @@ const messages = defineMessages({
   qualityprofile: 'Quality Profile',
   rootfolder: 'Root Folder',
   animenote: '* This series is an anime.',
-  default: '(Default)',
-  loadingprofiles: 'Loading profiles…',
-  loadingfolders: 'Loading folders…',
+  default: ' (Default)',
```

Comment:
We should leave formatting like this out of strings.

Reply:
It's not formatting, however; it's actually a part of the string.  And in some languages (like Chinese), not having this be a part of the string would result in whitespace where there shouldn't be any.

Reply:
But then you can argue that default shouldn't be a separate string at all? I think there can be some compromise here. I don't really like having spacing like this decided by the translation strings.

It would be better to have a string that is for default that takes the default value as variable.

"{value} (Default)" for example.

Reply:
Yes, I definitely think that's the better option! I thought about doing that, but figured it might be easier for our translators to just edit the space.  Will fix shortly!